### PR TITLE
Uses `XDG_CACHE_HOME` for `since` cache.

### PR DESCRIPTION
**Pull request by solomon-b**
Moves `since` cache from `/tmp` to `XDG_CACHE_HOME`.

### CODE REVIEW

**Comment by masaeedu** [comment]
Should we maybe use [this lib](https://hackage.haskell.org/package/xdg-basedir) so we don't have to deal with default locations or env var lookup ourselves?

**Comment by solomon-b** [comment]
> Should we maybe use [this lib](https://hackage.haskell.org/package/xdg-basedir) so we don't have to deal with default locations or env var lookup ourselves?

That lib hasn't been updated since 2012. You think its ok to use?

**Comment by masaeedu** [comment]
If the spec hasn't changed since then, I'd say yeah, probably?

**Comment by solomon-b** [comment]
Alright, I'll add it.

File: app/Main.hs [hunk 1]
```diff
@@ -8,31 +7,38 @@ import Data.Text qualified as T
...
 - [ ] Test suite
 - [ ] Administrative interface (via private message?)
 - [ ] Command to list all sessions
 - [ ] Add fixed point of Bot
+- [ ] Bot should auto join DMs
 -}
 
+findXdgCache :: IO String
+findXdgCache = do
+  home <- getEnv "HOME"
```
Comment (masaeedu) [inline app/Main.hs]: Is this still used?

Comment (solomon-b) [reply app/Main.hs]: oops

**Pull request merged**